ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/CompactMessage.qml

```diff
@@ -93,21 +95,76 @@ Item {
 
         UsernameLabel {
             id: chatName
-            visible: isMessage && headerRepeatCondition
+            visible: !isEdit && isMessage && headerRepeatCondition
             anchors.leftMargin: root.chatHorizontalPadding
             anchors.top: chatImage.top
             anchors.left: chatImage.right
         }
 
         ChatTime {
             id: chatTime
-            visible: headerRepeatCondition
+            visible: !isEdit && headerRepeatCondition
             anchors.verticalCenter: chatName.verticalCenter
             anchors.left: chatName.right
             anchors.leftMargin: 4
             color: Style.current.secondaryText
         }
 
+        Loader {
+            id: editMessageLoader
+            active: isEdit
+            anchors.top: chatReply.active ? chatReply.bottom : parent.top
+            anchors.left: chatImage.right
+            anchors.leftMargin: root.chatHorizontalPadding
+            anchors.right: parent.right
+            anchors.rightMargin: root.chatHorizontalPadding
+            height: (item !== null && typeof(item)!== 'undefined')? item.height: 0
```

Comment:
I'm not sure if the `height` line is really needed to be honest, I think it does it by default, but I myself have added that thing before, so I'm kinda lost

Also, double check my edit, maybe it doesn't work, but it should
```suggestion
            height: active ? item.height: 0
```